backend/src/bundles/avatars/config/config.ts

```diff
@@ -1,5 +1,8 @@
 import avatars from './avatar.config.json';
 
+// Removed styles "technical-standing" and "graceful-standing" from "lisa" avatar
+// due to broken images and insufficient config
+
```

Comment:
Please remove it

Reply:
@nikita-remeslov said that I should leave a comment why I deleted some avatars